avm/res/compute/disk-encryption-set/modules/nested_keyVaultPermissions.bicep

```diff
@@ -0,0 +1,69 @@
+@description('Required. A boolean to specify whether or not the used Key Vault has RBAC authentication enabled or not.')
+param rbacAuthorizationEnabled bool = true
+
+@description('Required. The resourceID of the User Assigned Identity to assign permissions to.')
+param userAssignedIdentityResourceId string
+
+@description('Optional. Resource location.')
+param location string = resourceGroup().location
+
+@description('Required. Resource ID of the KeyVault containing the key or secret.')
+param keyVaultResourceId string
+
+@description('Required. Key URL (with version) pointing to a key or secret in KeyVault.')
```

Comment:
I'd suggest to add a parameter for the image version (similar to the interface for CMK) so that we can fetch the image version for the user if not provided

Reply:
It is already implemented in the `main.bicep`. This nested bicep module only sets the permissions and we don't need the key version here as the permissions are set on the Key Vault level (access policy model) or on key level (RBAC model), but not for the particular key version,

Reply:
But I have updated the description of this parameter, as it was referring to an URL: https://github.com/Azure/bicep-registry-modules/pull/749/commits/ccb04d56f6138cd91c6bea79ff8aa47f80844c2f

Reply:
That certainly helps. Thanks